modules/ref-inference-endpoints.adoc

```diff
@@ -0,0 +1,218 @@
++
+--
+.REST endpoints
+
+* `/api/v1/task/embedding`
+* `/api/v1/task/embedding-tasks`
+* `/api/v1/task/sentence-similarity`
+* `/api/v1/task/sentence-similarity-tasks`
+* `/api/v1/task/rerank`
+* `/api/v1/task/rerank-tasks`
+* `/info/models`
+* `/info/version`
+* `/info/runtime`
+--
++
+--
+.gRPC endpoints
+
+* `:443 caikit.runtime.Nlp.NlpService/EmbeddingTaskPredict`
+* `:443 caikit.runtime.Nlp.NlpService/EmbeddingTasksPredict`
+* `:443 caikit.runtime.Nlp.NlpService/SentenceSimilarityTaskPredict`
+* `:443 caikit.runtime.Nlp.NlpService/SentenceSimilarityTasksPredict`
+* `:443 caikit.runtime.Nlp.NlpService/RerankTaskPredict`
+* `:443 caikit.runtime.Nlp.NlpService/RerankTasksPredict`
+* `:443 caikit.runtime.info.InfoService/GetModelsInfo`
+* `:443 caikit.runtime.info.InfoService/GetRuntimeInfo`
+--
+
+ifdef::upstream[]
+[NOTE]
+--
+By default, the Caikit Standalone Runtime exposes REST endpoints. To use gRPC protocol, manually deploy a custom Caikit Standalone ServingRuntime. For more information, see link:{odhdocshome}/serving-models/#adding-a-custom-model-serving-runtime-for-the-single-model-serving-platform_serving-large-models[Adding a custom model-serving runtime for the single-model serving platform]. 
+--
+
+An example manifest is available in the link:https://github.com/opendatahub-io/caikit-tgis-serving/blob/main/demo/kserve/custom-manifests/caikit/caikit-standalone/caikit-standalone-servingruntime-grpc.yaml[caikit-tgis-serving GitHub repository^].
+endif::[]
+
+ifndef::upstream[]
+[NOTE]
+--
+By default, the Caikit Standalone Runtime exposes REST endpoints. To use gRPC protocol, manually deploy a custom Caikit Standalone ServingRuntime. For more information, see link:{rhoaidocshome}{default-format-url}/serving_models/serving-large-models_serving-large-models#adding-a-custom-model-serving-runtime-for-the-single-model-serving-platform_serving-large-models[Adding a custom model-serving runtime for the single-model serving platform].
+--
+
+An example manifest is available in the link:https://github.com/opendatahub-io/caikit-tgis-serving/blob/main/demo/kserve/custom-manifests/caikit/caikit-standalone/caikit-standalone-servingruntime-grpc.yaml[caikit-tgis-serving GitHub repository^].
+endif::[]
+
+TGIS Standalone ServingRuntime for KServe::
+
+* `:443 fmaas.GenerationService/Generate`
+* `:443 fmaas.GenerationService/GenerateStream`
++
+[NOTE]
+--
+To query the endpoint for the TGIS standalone runtime, you must also download the files in the link:https://github.com/opendatahub-io/text-generation-inference/blob/main/proto[proto^] directory of the {productname-short} `text-generation-inference` repository.
+--
+
+OpenVINO Model Server::
+
+* `/v2/models/<model-name>/infer`
+
+vLLM ServingRuntime for KServe::
+
+* `:443/version`
+* `:443/docs`
+* `:443/v1/models`
+* `:443/v1/chat/completions`
+* `:443/v1/completions`
+* `:443/v1/embeddings`
++
```

Comment:
We need to add tokenize and detokenize endpoint as well . reference : https://github.com/vllm-project/vllm/blob/main/vllm/entrypoints/openai/api_server.py#L165-L176 

` /tokenize, Methods: POST
: /detokenize, Methods: POST```

Reply:
Do you all agree with this? @dtrifiro @terrytangyuan @christinaexyou

Reply:
Yes, tokenize/detokenize endpoints were added in [v0.5.1](https://github.com/vllm-project/vllm/releases/tag/v0.5.1),  `#5054`, `https://github.com/vllm-project/vllm/pull/5054`